config/README.rst

```diff
@@ -98,7 +98,7 @@ Preparation
 
 The script needs a ``venv`` with some packages installed::
 
-    $ python3 -m venv .
+    $ python3.11 -m venv .
```

Comment:
Should the newly created venv get activated?

Reply:
I never do.